tests/dapr_tests.mk

```diff
@@ -269,6 +277,7 @@ setup-test-components: setup-app-configurations
 	$(KUBECTL) apply -f ./tests/config/kubernetes_secret.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/kubernetes_secret_config.yaml --namespace $(DAPR_TEST_NAMESPACE)
 	$(KUBECTL) apply -f ./tests/config/kubernetes_redis_secret.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/dapr_mongodb_state.yaml --namespace $(DAPR_TEST_NAMESPACE)
```

Comment:
Rename to dapr_mongodb_querystate.yaml and have one as dapr_cosmosdb_querystate.yaml and use the environment variable to pick one, similar to how we do for state store. This will allow AKS to validate this with CosmosDB.

Reply:
it is still a regular statestore.